uisdk/src/main/java/com/karhoo/uisdk/screen/booking/quotes/filterview/QuoteTypesFilter.kt

```diff
@@ -10,11 +10,13 @@ class QuoteTypesFilter (selectedTypes: ArrayList<MultiSelectData>) : MultiSelect
         }
         return (quote.quoteType.ordinal == 0 && selectedTypes.any { it.fixedTag?.contains(FIXED_TAG) == true })
                 ||
-                (quote.quoteType.ordinal == 1 && selectedTypes.any { it.fixedTag?.contains(ESTIMATED_TAG) == true })
+                ((quote.quoteType.ordinal == 1 || quote.quoteType.ordinal == 2) && selectedTypes.any { it.fixedTag?.contains(ESTIMATED_TAG) == true || it.fixedTag?.contains(
```

Comment:
I don't like the fact that we're comparing ordinals. 0, 1, and 2 are not obvious enough as to what they represent. Also, the enums used for filtering are an internal convention. We can choose to reorder them for consistency between platforms which will mess up this logic. Please compare with the label, not the ordinal wherever it is the case. e.g. of the expected outcome: `quote.quoteType == QuoteType.FIXED`